MEDO v0.5.owl

```diff
@@ -586,7 +757,7 @@ LCME Glossary of Terms for LCME Accreditation Standards and Elements 2015-2016,
         <rdfs:subClassOf rdf:resource="http://purl.obolibrary.org/obo/medo.owl#MEDO_1000"/>
         <obo:IAO_0000115>a type of assessment where students must respond to some prompt or question, and the response is evaluated by the instructor.</obo:IAO_0000115>
```

Comment:
I like using "in which" instead of "where":  "A type of assessment in which students must respond to some prompt or question, and the response is evaluated by the instructor."  Also, remember to capitalize first word of all definitions.